ngraph/frontend/onnx_import/src/op/global_max_pool.cpp

```diff
@@ -41,11 +28,22 @@ namespace ngraph
                     // Input shape: [N, C, H, W, D]
                     // Input spatial dimensions are H, W and D
                     // Expected spatial dims indexes: [2, 3, 4]
-                    size_t data_spatial_rank = data_rank_value - 2;
-                    auto reduce_axes_vector = std::vector<std::int64_t>(data_spatial_rank);
-                    std::iota(reduce_axes_vector.begin(), reduce_axes_vector.end(), 2);
-                    auto reduce_axes = default_opset::Constant::create(
-                        element::i64, Shape{data_spatial_rank}, reduce_axes_vector);
+                    auto data = node.get_ng_inputs()[0];
```

Comment:
Do we need to have more tests?

Reply:
I don't think so, this is the same behavior as MO and ONNX tests should cover it.